Modules/Forum/classes/class.ilForumTopicTableGUI.php

```diff
@@ -426,7 +419,7 @@ public function fetchData()
 			$data = $this->getMapper()->getAllThreads($this->topicData['top_pk'], $params, (int)$this->getLimit(), (int)$this->getOffset());
 		}
 
-		$this->setMaxCount($data['cnt']);
+		$this->setMaxCount(count($data['items']));
```

Comment:
As I said in #2068 I am not sure if this will work. You cannot count the items manually, because this only includes the items presented at this table page. You'll have to count all possible items matching the current cireteria (filter).